st2common/st2common/models/db/execution.py

```diff
@@ -123,49 +122,39 @@ def get_masked_parameters(self):
         return serializable_dict['parameters']
 
 
-class ActionExecutionStdoutOutputDB(stormbase.StormFoundationDB):
+class ActionExecutionOutputDB(stormbase.StormFoundationDB):
     """
-    Stores stdout output of a particular action.
-
-    New document is inserted dynamically when a new line is received which means you can simulate
-    tail behavior by periodically reading from this collection.
-    """
-    execution_id = me.StringField(required=True)
-    action_ref = me.StringField(required=True)
-    timestamp = me.DateTimeField(required=True, default=date_utils.get_datetime_utc_now)
-
-    line = me.StringField()
-
-    meta = {
-        'indexes': [
-            {'fields': ['execution_id']},
-            {'fields': ['timestamp']},
-            {'fields': ['action_ref']}
-        ]
-    }
-
-
-class ActionExecutionStderrOutputDB(stormbase.StormFoundationDB):
-    """
-    Stores stderr output of a particular action.
-
-    New document is inserted dynamically when a new line is received which means you can simulate
-    tail behavior by periodically reading from this collection.
+    Stores output of a particular execution.
+
+    New document is inserted dynamically when a new chunk / line is received which means you can
```

Comment:
From this, I gather that this model just represents one "write"? Meaning, for each instance of data that an execution wishes to stream about itself, each would be a unique instance of this model?

Reply:
Correct - mentioned the reason for that in the other PR (performance - updating single document is not a good idea because of how locking works).